src/containers/GifGalleryContainer.js

```diff
@@ -0,0 +1,23 @@
+import GifGallery from "../components/GifGallery";
+import {fetchTrendingGifs} from "../actions/trending";
+import {connect} from "react-redux";
+import {fetchSearchGifs} from "../actions/search";
+import {updateFilteredFavorites} from "../actions/favorite";
+
+const mapStateToProps = ({favorites, trending}) => ({
+  favorites,
+  trending
+});
+
+const mapDispatchToProps = (dispatch) => ({
+  fetchTrendingGifs: () => dispatch(fetchTrendingGifs()),
+  fetchSearchGifs: (toSearch) => dispatch(fetchSearchGifs(toSearch)),
+  updateFilteredFavorites: (filteredFavorites) => dispatch(updateFilteredFavorites(filteredFavorites))
+});
```

Comment:
You can use the `bindActionCreators` util from 'redux' to avoid declaring these functions:

```js
const mapDispatchToProps = (dispatch) => bindActionCreators({
  fetchTrendingGifs,
  fetchSearchGifs,
  updateFilteredFavorites,
});
```